sygnal/apnspushkin.py

```diff
@@ -172,31 +172,34 @@ def __init__(self, name: str, sygnal: "Sygnal", config: Dict[str, Any]) -> None:
             # this overrides the create_connection method to use a HTTP proxy
             loop = ProxyingEventLoopWrapper(loop, proxy_url_str)  # type: ignore
 
-        if certfile is not None:
-            # max_connection_attempts is actually the maximum number of
-            # additional connection attempts, so =0 means try once only
-            # (we will retry at a higher level so not worth doing more here)
-            self.apns_client = APNs(
-                client_cert=certfile,
-                use_sandbox=self.use_sandbox,
-                max_connection_attempts=0,
-                loop=loop,
-            )
+        async def make_apns() -> aioapns.APNs:
+            if certfile is not None:
+                # max_connection_attempts is actually the maximum number of
+                # additional connection attempts, so =0 means try once only
+                # (we will retry at a higher level so not worth doing more here)
+                apns_client = self.apns_client = APNs(
```

Comment:
this seems weird, since we are setting `self.apns_client` with the return value, so this should probably just be

```
apns_client = APNs(
```

Reply:
Whoops, bad copy paste